sqlite_tests/litefs.yml

```diff
@@ -0,0 +1,92 @@
+# The FUSE section handles settings on the FUSE file system. FUSE
+# provides a layer for intercepting SQLite transactions on the
+# primary node so they can be shipped to replica nodes transparently.
+fuse:
+  # Required. This is the mount directory that applications will
+  # use to access their SQLite databases.
+  # Use non-root path as sqlite tests can't run as root
+  dir: "/home/build/litefs"
```

Comment:
Since we need to run tests as the `build` user, easier to just have all `litefs` paths be owned by `build` user